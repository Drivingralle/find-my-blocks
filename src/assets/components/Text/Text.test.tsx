import React from "react";
import renderer from "react-test-renderer";
import { Text } from ".";

it("renders the Text", () => {
  const tree = renderer.create(<Text>Hello</Text>).toJSON;
  expect(tree).toMatchSnapshot();
});
