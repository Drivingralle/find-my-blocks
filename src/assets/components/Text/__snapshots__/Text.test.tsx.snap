// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Text 1`] = `[Function]`;
